### DISCUSSION TITLE
Marking calculated columns in the Data Preview table

### DISCUSSION THREAD

**Discussion by Stewwe1** [TabularEditor/TabularEditor3#1124] [Ideas]
Hello TE3 Team,

I don't know if anyone has suggested this before, but we would like it very much if Calculated Columns were marked in the "Preview Data table", e.g. with an icon like in the TOM Explorer or a slightly different color.

Simply increases the clarity :-)

Kind regards

Steffen

**Comment by mlonsk** [comment]
Hi @Stewwe1 
Thank you for the suggestion, sounds like a good idea to have this marked in some way.

**Comment by otykier** [comment]
Hello @Stewwe1 - happy to let you know that we now, finally, have made this improvement in [3.14.0](https://docs.tabulareditor.com/te3/other/release-notes/3_14_0.html?tabs=downloads):

<img width="571" alt="image" src="https://github.com/TabularEditor/TabularEditor3/assets/8976200/162921e1-db18-4bf6-96b6-599939c6244e">
